src/wallet/rpcdump.cpp

```diff
@@ -573,7 +582,10 @@ UniValue importwallet(const JSONRPCRequest& request)
                 },
             }.ToString());
 
-    if (pwallet->chain().getPruneMode()) {
+    if (pwallet->chain().havePruned()) {
+        // Exit early and print an error.
+        // If a block is pruned after this check, we will import the key(s),
+        // but fail the rescan with a generic error.
         throw JSONRPCError(RPC_WALLET_ERROR, "Importing wallets is disabled in pruned mode");
```

Comment:
Does the code doc in 588 correspond correctly with the error message in 589?

Reply:
Yes, "importing wallet" means "importing key(s)", I believe.